src/components/date-picker/date-picker-dropdown.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
```

Comment:
I would love to see some universal `<Dropdown/>` implementation and then its usage in `<DatePickerDropdown/>`